tests/test_free_energy_rabfe.py

```diff
@@ -31,7 +31,17 @@ def capture_print(closure):
 def test_rabfe_result_to_from_log():
     """assert equality after round-trip to/from preferred terminal log format"""
 
-    result = RABFEResult("my mol", 1.0, 2.0, 3.0, 4.0)
+    result = RABFEResult(
+        "my mol",
+        1.0,
+        0.0,
+        2.0,
+        0.0,
+        3.0,
+        0.0,
+        4.0,
+        0.0,
```

Comment:
Might as well make all 8 values distinct

Reply:
Also added a nan as I know @proteneer had wanted to know if they round tripped

Reply:
Nice -- resolved in https://github.com/proteneer/timemachine/pull/566/commits/920806dbfd283f2fbfa499dd3ad8d011a159037c and https://github.com/proteneer/timemachine/pull/566/commits/08ce6bf4a335f69498f03bd17a1b99d7f912510c